### DISCUSSION TITLE
NO more features

### DISCUSSION THREAD

**Discussion by ghost** [mhaeuser/Battery-Toolkit#50] [General]
Thank you for all your hard work on this project. We don’t need any more features. What made me switch from Aldente is your app’s simplicity and efficiency. It’s small, optimized, and works great. Thanks again!

**Comment by mhaeuser** [comment]
Honestly, the only technical reason I even wrote it was that AlDente used polling over events. As I didn’t want to pay for polling too, I just quickly wrote this. I didn’t plan adding features beyond refining the UX (stateful menu bar extra, maybe refinements to the MagSafe light stuff, etc) anyway.
